ui/src/app/pages/profile/data-use-agreement-content.tsx

```diff
@@ -0,0 +1,200 @@
+import {BolderHeader, BoldHeader} from 'app/components/headers';
+import * as React from 'react';
+import {IndentedListItem, IndentedOrderedList, IndentedUnorderedList, SecondHeader} from './data-use-agreement';
+
+const CORE_VALUES_URL = 'https://allofus.nih.gov/about/about-all-us-research-program';
+
+const AoUTitle = () => {
+  return <span><i>All of Us</i> Research Program</span>;
+};
+
+{/* NOTE: Make sure to update dataUseAgreementVersion if there is any change to the DUA text. */}
+export const DataUseAgreementContent = () => {
+  return <div>
+    <BolderHeader style={{display: 'flex', justifyContent: 'center'}}><AoUTitle/></BolderHeader>
+    <BoldHeader  style={{display: 'flex', justifyContent: 'center'}}>Demonstration Project Data Use Agreement</BoldHeader>
+    <p>
+      This data use agreement describes how <AoUTitle/> data can and cannot be used for the purposes
+      of program-approved demonstration projects.
+    </p>
+    <p>
+      This is an agreement between Vanderbilt University Medical Center and authorized demonstration users of data
+      from the <AoUTitle/>.
+    </p>
+    <p>
+      An <strong>authorized demonstration user</strong> is a person who is authorized to access
+      and/or work with registered or
+      controlled tier data from the <AoUTitle/> for the exclusive purpose of a program-approved
+      demonstration project. Authorized demonstration users are limited to trainees, faculty or staff at <AoUTitle/>
+      consortium partner institutions.
+    </p>
+    <p>
+      <strong>Before</strong> they access and/or work with <AoUTitle/> data, authorized demonstration users must:
+      <IndentedOrderedList>
+        <li>complete the <AoUTitle/> research ethics training; and</li>
+        <li>read and attest to this data use agreement</li>
+      </IndentedOrderedList>
+    </p>
+    <p>Please read this agreement carefully and completely before signing.</p>
+    <SecondHeader>As “Authorized Demonstration User” of the <AoUTitle/> data, I
+      will:</SecondHeader>
+    <IndentedUnorderedList>
+      <li>read and adhere to the <AoUTitle/> <a target='_blank' href={CORE_VALUES_URL}>core values</a>.
+      </li>
+      <li>know and follow all applicable US federal, state, and local
+        (applicable in the area where I am conducting research) laws
+        regarding research involving human data and data privacy.
+      </li>
+      <li>respect the privacy of research participants at all times.
+        <IndentedUnorderedList>
+          <IndentedListItem style={{margin: '0.5rem 0'}}>
+            I will <strong>NOT</strong> use or disclose any
+            information that directly identifies one or more participants.
+            <br/>
+            If I become aware of any information that directly identifies one or more
+            participants, I will notify the <AoUTitle/> immediately
+            using the automatic notification system.
```

Comment:
This says `automatic notification system` but the doc says `using the appropriate process.`? (I like yours better, but it is a diff against the doc)

Reply:
Good catch. That might have been one place where the old doc, new doc, and html in code was different...